src/app/component/user/components/habit/all-habits/components/habits-list-view/habits-list-view.component.scss

```diff
@@ -1,5 +1,211 @@
-.card {
-  width: 100%;
-  height: 200px;
-  border: 1px solid black;
+@import url("https://fonts.googleapis.com/css?family=Open+Sans&display=swap");
+@import url("https://fonts.googleapis.com/css?family=Montserrat&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+.list-habit-content {
+  display: flex;
+  height: 296px;
+  width: 944px;
+  margin-bottom: 30px;
+  border-radius: 8px;
+  background: #f5f6f6 url(/assets/img/icon/HabitPicture.png) no-repeat  top 20px right -61px/34%;
+
+  .header {
+    display: flex;
+    text-transform: uppercase;
+    padding: 35px 0 15px 35px;
+    font-family: "OpenSans", sans-serif;
+    font-size: 14px;
+    line-height: 16px;
+    color: #13aa57;
+
+    .calendar {
+      .calendar__icon {
+        margin-top: -3px;
```

Comment:
-3px isn't a good way to write the styles. Rewrite, please

Reply:
Fixed